tests/unit/test_video_rest.py

```diff
@@ -403,8 +401,8 @@ def test_video_publish_edit_publish_again(
             # [[ MODIFY DOI -> SAVE ]]
             video_1 = deposit_video_resolver(video_1_depid)
             video_1_dict = copy.deepcopy(video_1)
-            old_doi = video_1_dict["doi"]
-            video_1_dict["doi"] = "10.1123/doi"
+            old_doi = video_1_dict.get("doi")
+            # video_1_dict["doi"] = "10.1123/doi"
```

Comment:
Minor: if not needed simply better to remove than to comment out, and there is no need to add the `.get`? or is it?

Reply:
Updated! Added DOI minting to the test, so it can be used as before